integration/watch_false/Tiltfile

```diff
@@ -0,0 +1,10 @@
+# -*- mode: Python -*-
+
+# If you get push errors, you can change the default_registry.
+# Create tilt_option.json with contents: {"default_registry": "gcr.io/my-personal-project"}
+# (with your registry inserted). tilt_option.json is gitignore'd, unlike Tiltfile
+default_registry(read_json('tilt_option.json', {})
+                 .get('default_registry', 'gcr.io/windmill-test-containers/servantes'))
```

Comment:
it seems weird to use servantes for this?

Reply:
ha it's copypasta from other integration tests